docs/gettinginvolved/cluster-setup.rst

```diff
@@ -199,6 +171,79 @@ Or you can run, for example, the acceptance tests against the created cluster:
      --flocker-version='' \
      flocker.acceptance.obsolete.test_containers.ContainerAPITests.test_create_container_with_ports
 
+==============================
```

Comment:
Or is it a sub-section? I've just noticed that some headings in this topic are classed as top level headers, so maybe the syntax here is correct? Your call @mpgomez